crates/matrix-sdk-base/src/sliding_sync.rs

```diff
@@ -64,7 +64,7 @@ impl BaseClient {
 
         let to_device = to_device.as_ref().map(|v4| v4.events.clone()).unwrap_or_default();
 
-        info!(
+        trace!(
```

Comment:
Let's add a message to this call, alternatively let's put all of those into an instrument macro on the method.